eclass/meson.eclass

```diff
@@ -125,6 +125,7 @@ _meson_create_cross_file() {
 	ar = '${AR}'
 	c = '${CC}'
 	cpp = '${CXX}'
+	pkgconfig = '${PKG_CONFIG}'
```

Comment:
This doesn't appear to be documented anywhere. Are you sure meson even looks at this setting?

Reply:
I recall looking into meson's source to find the correct variable name to use.  I can look around again for better insight.  Here is an example cross-file using it in the upstream meson repo: https://github.com/mesonbuild/meson/blob/master/cross/ubuntu-mingw.txt#L10 .  At any rate, this change does allow systemd to be configured.

Reply:
Ah, I see it does.

https://github.com/mesonbuild/meson/blob/master/mesonbuild/dependencies/base.py#L183